cli_tools/common/image/importer/api_inflater.go

```diff
@@ -125,20 +162,18 @@ func (inflater *apiInflater) Inflate() (persistentDisk, shadowTestFields, error)
 		inflationTime: time.Since(startTime),
 	}
 
-	// Calculate checksum by daisy workflow
-	inflater.logger.Debug("Started checksum calculation.")
-	ii.checksum, err = inflater.calculateChecksum(ctx, diskURI)
-	if err != nil {
-		err = daisy.Errf("Failed to calculate checksum: %v", err)
-	}
-	return pd, ii, err
+	return pd, ii, nil
 }
 
 func (inflater *apiInflater) getShadowDiskName() string {
 	return fmt.Sprintf("shadow-disk-%v", inflater.request.ExecutionID)
 }
 
 func (inflater *apiInflater) Cancel(reason string) bool {
+	if !inflater.isShadowInflater {
```

Comment:
Can you add a comment why this is always false?

Reply:
done